backend/ohq/statistics.py

```diff
@@ -232,3 +234,83 @@ def queue_calculate_questions_per_ta_heatmap(queue, weekday, hour):
         hour=hour,
         defaults={"value": statistic if statistic else 0},
     )
+
+
+def user_calculate_questions_asked(user):
+    num_questions = Question.objects.filter(asked_by=user).count()
+
+    if num_questions and not num_questions == 0:
+        UserStatistic.objects.update_or_create(
+            user=user,
+            metric=UserStatistic.METRIC_TOTAL_QUESTIONS_ASKED,
+            defaults={"value": num_questions},
+        )
+
+
+def user_calculate_questions_answered(user):
+    num_questions = (Question.objects.filter(
+        responded_to_by=user,
+        status=Question.STATUS_ANSWERED
+        ).count())
+
+    if num_questions and not num_questions == 0:
+        UserStatistic.objects.update_or_create(
+            user=user,
+            metric=UserStatistic.METRIC_TOTAL_QUESTIONS_ANSWERED,
+            defaults={"value": num_questions},
+        )
+
+
+
+def user_calculate_time_helped(user):
+    user_time_helped = (
+        Question.objects.filter(
+            asked_by=user,
+            status=Question.STATUS_ANSWERED
+            ).aggregate(time_helped=Sum(F("time_responded_to") - F("time_response_started")))
+    )
+    time = user_time_helped["time_helped"]
+
+    if time and not math.isclose(time.total_seconds(), 0, abs_tol=0.001):
```

Comment:
Question: is there a reason you use `time.total_seconds()` here, but a few lines down you use `time.seconds`? Is there a difference?

Reply:
lol nope I don't think there's a difference I was just copying off of the documentation- I'll change it to .seconds for consistency